Arrowgene.Ddon.Database/Files/Database/Script/schema_sqlite.sql

```diff
@@ -496,3 +496,16 @@ CREATE TABLE IF NOT EXISTS "ddon_bazaar_exhibition" (
     "expire"          DATETIME                          NOT NULL,
     FOREIGN KEY("character_id") REFERENCES "ddon_character"("character_id")
 );
+
+CREATE TABLE IF NOT EXISTS "ddon_reward_box" (
+	"uniq_reward_id"	INTEGER NOT NULL,
+	"character_common_id"	INTEGER NOT NULL,
```

Comment:
Since pawns can't get quest rewards i think it's more appropriate to have it be character_id instead, but its fine either way